### PR TITLE
feat: add bulk import component

### PR DESCRIPTION
**Pull request by punwai**
## Description
- Add bulk import component
- Implemented module loaded with lazy loading

## Related issues and discussion
#542

## Screenshots, if any
Screenshot 2020-01-10 at 13 28 39

## Checklist
Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] If you have multiple commits please combine them into one commit by squashing them.

- [x] Read and understood the contribution guidelines at `web-app/.github/CONTRIBUTING.md`.

### CODE REVIEW

**Comment by punwai** [comment]
Fail due to time out.

File: src/app/bulk-import/bulk-import.module.ts [hunk 1]
```diff
@@ -0,0 +1,25 @@
+/** Angular Imports */
+import { NgModule } from '@angular/core';
+
+/** Custom Modules */
+import { SharedModule } from '../shared/shared.module'
+import { BulkImportRoutingModule } from './bulk-import-routing.module'
```
Comment (abhaychawla) [inline src/app/bulk-import/bulk-import.module.ts]: https://travis-ci.com/openMF/web-app/builds/143895840

The Travis CI build is failing due to lint errors.

File: src/app/organization/organization-routing.module.ts [hunk 2]
```diff
@@ -197,6 +197,10 @@ const routes: Routes = [
           resolve: {
             workingDays: WorkingDaysResolver
           }
+        },
+        {
+          path: 'bulk-import',
+          loadChildren: '../bulk-import/bulk-import.module#BulkImportModule'
```
Comment (abhaychawla) [inline src/app/organization/organization-routing.module.ts]: Really appreciate your willingness to learn!!

File: src/app/bulk-import/bulk-import.component.ts [hunk 3]
```diff
@@ -0,0 +1,19 @@
+/** Angular Imports */
+import { Component, OnInit } from '@angular/core';
+
+/**
+ * Accounting component.
+ */
```
Comment (abhaychawla) [inline src/app/bulk-import/bulk-import.component.ts]: Incorrect comment.

File: src/app/organization/organization-routing.module.ts (hunk 2 above)
Comment (punwai) [reply src/app/organization/organization-routing.module.ts]: @abhaychawla Thank you :)

**Comment by punwai** [comment]
I apologise about the build error that I said it was linting. I linted locally and fixed the issues, but I forgot to push out my last rebased commit so I just clumsily assumed that it was due to timeout without actually checking the log.

**Comment by punwai** [comment]
@abhaychawla If you do have any harder components in mind, I would be delighted to do it as a task if you set it, I would love to challenge myself abit more on what I could implement.

**Comment by abhaychawla** [comment]
Alright, let me see if we can add some more relatively challenging tasks.

**Comment by punwai** [comment]
@abhaychawla Sorry to disturb you, I was just wondering whether you have had time to consider about the relatively challenging tasks. I understand that you are very busy, so please don't feel pressured to do this, I really appreciate what you are currently doing.

**Comment by punwai** [comment]
@abhaychawla Of course its entirely up to you, but I'm just saying that I would be happy to work on components such as #256. It would really be nice to work on something challenging and different, and it would really be a great learning experience for me.

**Comment by abhaychawla** [comment]
@punwai Let me try to get some challenging tasks published by tomorrow. I have been actually really busy with office work the past couple of days due to which I was unable to give it much thought.

**Comment by punwai** [comment]
@abhaychawla Don't worry, you are not doing this out of any obligations and I really appreciate that. Thank you very very much.

**Pull request merged**